version.gradle.kts

```diff
@@ -24,8 +24,9 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-val spineBaseVersion: String by extra("2.0.0-SNAPSHOT.75")
-val spineBaseTypesVersion: String by extra("2.0.0-SNAPSHOT.75")
-val spineCoreVersion: String by extra("2.0.0-SNAPSHOT.89")
-val mcJavaVersion: String by extra("2.0.0-SNAPSHOT.76")
-val versionToPublish: String by extra("2.0.0-SNAPSHOT.66")
+val spineBaseVersion: String by extra("2.0.0-SNAPSHOT.88")
+val spineBaseTypesVersion: String by extra("2.0.0-SNAPSHOT.88")
+val spineCoreVersion: String by extra("2.0.0-SNAPSHOT.94")
+val mcJavaVersion: String by extra("2.0.0-SNAPSHOT.83")
+
+val versionToPublish: String by extra("2.0.0-SNAPSHOT.67")
```

Comment:
I would go with `94` here, as these artifacts are closely related to `core-java`.